sample-docker-app/ax-manifest.yml

```diff
@@ -4,5 +4,6 @@ id: com.actyx.sample-docker-app
 version: 1.0.0
 displayName: Sample Docker App
 description: "A sample docker app"
-dockerCompose: ./docker-compose.yml
+dockerCompose: 
+  x86_64: ./docker-compose.yml
 settingsSchema: ./settings.schema.json
```

Comment:
This is a pet peeve of mine: Please configure your editors to add a newline at the end of the file